src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -77,12 +100,30 @@ export default {
             imageFormat: '',
             imageHistory: [],
             imageUrl: '',
+            age: '',
             isPaused: false,
             metadata: {},
             requestCount: 0,
-            timeFormat: ''
+            time: undefined,
+            timeFormat: '',
+            clock: clock,
+            ageTracker: undefined,
+            refreshCSS: false
         };
     },
+    computed: {
+        isImageNew() {
+            let cutoff = FIVE_MINUTES;
+            let age = this.numericImageAge();
+
+            return age < cutoff;
+        }
+    },
+    watch: {
+        time() {
+            this.resetAgeCSS();
```

Comment:
I think that this only needs to happen when the image changes. The code might be a little clearer if `resetAgeCSS` were called at the point at which the image changes.

Reply:
Moving this to updateValues as well